openslides/__main__.py

```diff
@@ -135,7 +135,7 @@ def start(args):
     execute_from_command_line(['manage.py', 'migrate'])
 
     if not args.no_browser:
-        start_browser('http://0.0.0.0:8000')
+        start_browser('http://127.0.0.1:8000')
```

Comment:
@normanjaeckel Can you find another source? I do not belief what that comment says, because I understood something else from the term "unix domain socket". And also, I think, that this would only make a difference, if the server listens to localhost, and not, that the client connets to localhost, what is mapped to 127.0.0.1 in /etc/hosts